recipes/ditaa/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ditaa" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stathissideris/ditaa/releases/download/v0.11.0/ditaa-0.11.0-standalone.jar
```

Comment:
Please template with the version variable here

Reply:
Updated in the recent commit